owa/app/js/subscription/subscription.controller.js

```diff
@@ -48,6 +49,22 @@ class SubscriptionController {
         vm.subscription = response.results[0];
       })
     }
+
+    function isVersionAdded(url) {
+      if (url.endsWith("/")) {
+        url = url.substring(0, url.lastIndexOf('/'));
+      }
+      let words = url.split("api.openconceptlab.org")
+
+      let count = words[1].length - words[1].replace(/[\/\\]/g, '').length;
+
+      if (count == 5) {
```

Comment:
What is the meaning of the magic number `5`?

Reply:
I'm using that while doing the string manipulations too, maybe I should have it declared someone, will that make any difference?
I mean not having the version URL section presented as a Param introduces all these codes

Reply:
A constant with a name which help me know why it is not 6 or 7, would be helpful. 😊

Reply:
I changed this value into a constant